src/backend/Dockerfile.nextstrain

```diff
@@ -65,4 +65,7 @@ COPY . .
 # Install the aspen package
 RUN poetry install
 
+# TODO - temporary workaround
```

Comment:
Would be nice to have an additional comment line saying why the workaround is necessary:
"Mismatch between poetry and augur deps forces us to manually install jsonschema v3 here" or similar